app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,22 @@
+module SessionsHelper
+  def log_in(user)
+    session[:user_id] = user.id
+  end
+
+  def remember(user)
+    cookies.permanent.signed[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by(id: session[:user_id]) if session[:user_id]
```

Comment:
There should be no application logic in controllers (for instance, no `Post.new`, `Comment.create`, `current_user.friends.find_by(...)`...)

See this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/section/no_logic_in_helpers_requirement.png) from Pathwright